sdk/canton/community/app/src/pack/examples/06-messaging/message/daml.yaml

```diff
@@ -1,6 +1,4 @@
-sdk-version: 2.10.0-snapshot.20241210.13134.0.vad20cd7e
```

Comment:
I dont think its worth updating the canton ones. I'm also not sure if changing the sdk version here will cause breakages.

Reply:
Hummm... without this change, the branch was in conflict and could not be merged. 🤔